docs/tutorial/part01.rst

```diff
@@ -2,5 +2,83 @@
 Set up the environment.
 =======================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+The following section will have the steps to set your local environment, so you can begin to work on your Django-neomodel implementation.
+If you want a point to start, we provide a base app to begin the tutorial. First clone the paradise-papers-django_ repo.
+
+    .. _paradise-papers-django: https://github.com/neo4j-examples/paradise-papers-django
+
+After you download the repo go to the ``start_app`` branch and you will get a working app that will be our start point.
+
+.. [*] Note:

```

Comment:
here the with the front-end implementation. is on another line, that was intentional? 
![image](https://user-images.githubusercontent.com/29434543/35441263-32b0867c-0279-11e8-9088-c0f2bfa9eea1.png)

Reply:
Nop was just an indentation mistake. Nice catch!!!!